PayloadPkg/OsLoader/LoadImage.c

```diff
@@ -182,12 +184,8 @@ GetBootImageFromRawPartition (
     //
     // Read the rest of the IAS image into the buffer
     //
-
-    if ( BootOption->DevType == OsBootDeviceMemory ) {
-      Address =  LogicBlkDev.StartBlock + LbaAddr + AlignedHeaderSize;
-    } else {
-      Address =  LbaAddr + AlignedHeaderSize;
-    }
+    AlignedHeaderBlkCnt = AlignedHeaderSize / BlockSize;

```

Comment:
Can we calculate at line AlignedHeaderBlkCnt  110 ?  In this way, some duplicated calculation can be removed.
I think it can be something like below:
  AlignedHeaderBlkCnt  = (sizeof (IAS_HEADER) + (BlockSize - 1)) / BlockSize;
  AlignedHeaderSize      = AlignedHeaderBlkCnt * BlockSize;